api/src/Page/Sample.php

```diff
@@ -1748,56 +1760,23 @@ function _proteins()
                                 HEX(pr.externalid) as externalid,
                                 pr.density,
                                 count(php.proteinid) as pdbs,
-                                pr.safetylevel
+                                pr.safetylevel,
+                                count(dc.datacollectionid) as dcount,
+                                count(b.blsampleid) as scount
 
                                 FROM protein pr
                                 LEFT OUTER JOIN concentrationtype ct ON ct.concentrationtypeid = pr.concentrationtypeid
                                 LEFT OUTER JOIN componenttype cmt ON cmt.componenttypeid = pr.componenttypeid
                                 LEFT OUTER JOIN protein_has_pdb php ON php.proteinid = pr.proteinid
-                                /*LEFT OUTER JOIN crystal cr ON cr.proteinid = pr.proteinid
+                                LEFT OUTER JOIN crystal cr ON cr.proteinid = pr.proteinid
                                 LEFT OUTER JOIN blsample b ON b.crystalid = cr.crystalid
-                                LEFT OUTER JOIN datacollection dc ON b.blsampleid = dc.blsampleid*/
+                                LEFT OUTER JOIN datacollection dc ON b.blsampleid = dc.blsampleid
```

Comment:
`BLSAMPLEID` as a column is possibly deprecated, could we perform this join in another way? Possibly through the `blSubSampleId` column, if that is not always populated

Also, could we perform said join as a series of inner joins, for performance reasons?